packages/bot-web-ui/src/app/app-content.jsx

```diff
@@ -68,14 +68,26 @@ const AppContent = observer(() => {
         ServerTime.init(common);
         app.setDBotEngineStores(combinedStore);
         ApiHelpers.setInstance(app.api_helpers_store);
-        const { active_symbols } = ApiHelpers.instance;
         setIsLoading(true);
-        active_symbols.retrieveActiveSymbols(true).then(() => {
-            setIsLoading(false);
-        });
+        const { active_symbols } = ApiHelpers.instance;
+        if (!client.is_logged_in) {
+            active_symbols.retrieveActiveSymbols(true).then(() => {
+                setIsLoading(false);
+            });
+        }
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
+    // use is_landing_company_loaded to know got details of accounts to identify should show an error or not
+    if (client.is_landing_company_loaded) {
+        const { active_symbols } = ApiHelpers.instance;
+        active_symbols?.retrieveActiveSymbols(true).then(() => {
+            if (client.is_landing_company_loaded) {
+                setIsLoading(false);
+            }
+        });
+    }
+
```

Comment:
As we are doing the same operation "active_symbols.retrieveActiveSymbols(true)" twice, is it possible to combine them and do it once? Or create another function for this operation and call it based on the condition?

Reply:
@shafin-deriv Actually in the lines 72-77 we call it one time when the user doesn't logged in and stop loader. 
In the second place (lines 81-89) we check that user logged in + we got data what kind of user with help `is_landing_company_loaded` from `core -client-store` and only then stop loader in order don't show blockly workspace before showing error per sec if we should(avoid flicker).
So for the second case happens when `is_landing_company_loaded` becomes `true`. 
I've tiny refactored with useEffect

Reply:
@shafin-deriv it was changed because broke a few cases, we should do it when it is only = true + once. it is like this.